app/_hub/kong-inc/request-transformer/index.md

```diff
@@ -56,78 +60,136 @@ params:
   config:
     - name: http_method
       required: false
+      datatype: string
```

Comment:
@murillopaula what are the supported methods? Any default? Nothing in schema or EE plugin either.

Reply:
here it is: https://github.com/Kong/kong/blob/master/kong/db/schema/typedefs.lua#L212-L215

the plugin just imports it with https://github.com/Kong/kong-plugin-request-transformer/blob/master/kong/plugins/request-transformer/schema.lua#L3

so, about the supported methods, the schema defines a `string` that must match the Lua regex `"^%u+$"` - the `^` is the anchor to the beginning of the string, the `%u` means "upper case letters", the `+` means "match one or more uppercase letters in the beginning of the string" and finally the `$` is the anchor to the end of the string, so after all the uppercase letters the string must end, nothing should come after the uppercase letters

Reply:
in summary, it's a regex for HTTP methods such as `POST`, `GET`, `PATCH`